site/community.md

```diff
@@ -49,9 +49,9 @@ ManageIQ is open source and open for contributions. There are many ways to get i
 
     <h3>Sprint Reviews</h3>
 
-    <p>There is a ManageIQ Sprint review every 2 weeks open to the community. To join the online video conference, please use this <a href="https://bluejeans.com/9173928448/">Bluejeans link</a>.</p>
+    <p>There is a ManageIQ Sprint review every 2 weeks open to the community. To join the online video conference, please use this <a href="https://meet.google.com/nwr-oyqu-ukh">Google Meet link</a>.</p>
 
-    <p>You can import the <a href="https://calendar.google.com/calendar/embed?src=contact%40manageiq.org">ManageIQ community calendar</a> to be notified about ManageIQ events and Sprint reviews.</p>
+    <p>You can import the <a href="https://calendar.google.com/calendar/embed?src=content.admin%40manageiq.org">ManageIQ community calendar</a> to be notified about ManageIQ events and Sprint reviews.</p>
```

Comment:
@Fryguy I am not sure that this needs to be changed as the current link from https://www.manageiq.org/community/ shows a calendar with both the Sprint Review and the Triage meeting.  Thoughts?

Reply:
If we want to update those calendar items, then contact@'s calendar is fine, but we may just want to kill those meetings on contact@ and moved them to content.admin@